backend_new/test/integration/listing.e2e-spec.ts

```diff
@@ -92,7 +82,7 @@ describe('Listing Controller Tests', () => {
       filter: [
         {
           $comparison: Compare.IN,
-          name: 'name: 10,name: 50',
+          name: 'name: 11,name: 51',
```

Comment:
Curious why this is 11 and 51?

Reply:
I took the changes in https://github.com/bloom-housing/bloom/pull/3478/files#diff-4846ef09a013a233b0132f82b567da6c19be50bb3001fb8d38b2474cb18605c4 to fix the broken test. I believe it is relatively random, but @YazeedLoonat might know better

Reply:
on line 53 of that file we created listings 10 and 50 and since we no longer clean up after each test, we needed to switch to 2 different numbers otherwise we would find 10 and 50 each time 

so we switched to 11 and 51

those numbers are all random but we did need to guarantee no collision

Reply:
@YazeedLoonat was the uuid stuff we paired on wednesday a better solution to this?

Reply:
@YazeedLoonat Checking in here with you!

Reply:
good question! We made those UUID changes for unit tests where we mock interactions with the db

in e2e tests we are creating entries in the database. If we were just using IDs then we could have created another set of 50 and 10, but 
```
$comparison: Compare.IN,
name: 'name: 11,name: 51',
```
would mean that we find 4 records (2 50s and 2 10s) 

I'm not sure if that explanation makes sense, hopefully that helps though!